File: Programming Basics with Python/По-сложни проверки/cinema.py

ticket_type = input()
rows = int(input())
colums = int(input())

cinema_seats = rows * colums

if ticket_type == "Premiere":
    total_price = cinema_seats * 12
elif ticket_type == "Normal":
    total_price = cinema_seats * 7.50
elif ticket_type == "Discount":
    total_price = cinema_seats * 5

print(f"{total_price:.2f}")

File: Programming Basics with Java/More complex checks/Cinema.java
import java.util.Scanner;

public class Cinema 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);

        String ticket_type = input.nextLine();
        int row = input.nextInt();
        int column = input.nextInt();

        double ticket_price = 0.0;

        switch (ticket_type)
        {
            case "Premiere":
            {
                ticket_price = 12;
                break;
            }
            case "Normal":
            {
                ticket_price = 7.5;
                break;
            }
            case "Discount":
            {
                ticket_price = 5;
                break;
            }
        }

        double price = ticket_price * row * column;
        System.out.printf("%.2f leva", price);
    }
}
